tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -29,6 +29,8 @@ type PSATestSuite struct {
 	client              *rancher.Client
 	nonAdminUser        *management.User
 	nonAdminUserClient  *rancher.Client
+	standardUser        *management.User
+	standardUserClient  *rancher.Client
```

Comment:
we seem to have a nonAdminUser, nonAdminUserClient and standardUser, standardUserClient. Can we remove standardUser and use nonAdminUser as we are using this naming for restricted admin too?